apps/web/components/molecules/comments/comments.mock.tsx

```diff
@@ -0,0 +1,39 @@
+import type { Comment } from '../../../types/types';
+
+export const commentsMock: Comment[] = [
+	{
+		author: 'Adam Nowak',
+		comment:
+			'To zadanie frontendowe wydaje się bardzo wymagające, ale na pewno pozwoli na rozwinięcie umiejętności i poszerzenie wiedzy. Trzeba będzie zwrócić uwagę na każdy szczegół, aby stworzyć estetyczny i responsywny interfejs.',
+		avatar:
+			'https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2960&q=80',
+		date: 1264938840000,
+	},
+
+	{
+		author: 'Maria Wójcik',
+		comment:
+			'Mam nadzieję, że w tym zadaniu będzie można wykorzystać najnowsze technologie i narzędzia, co pozwoli na jeszcze lepsze efekty. Czekam na wyzwania, które pozwolą mi się rozwijać i poszerzać horyzonty.',
+		avatar:
+			'https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2960&q=80',
```

Comment:
Maybe some other photo for other mocked users? 😄 Would bring some more live and variety to the app during the development with nearly zero cost 😉